### PR TITLE
final class method and variable created, static block method and vari…

### PR DESCRIPTION
**Pull request by csestak-solvd**
…able created, 5 interfaces/classes created

### CODE REVIEW

File: src/interfaces/RestroomAvailability.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package interfaces;
+
+public abstract class RestroomAvailability {
```
Comment (ankochergin) [inline src/interfaces/RestroomAvailability.java]: why it's an abstract class? I think this should be an interface

File: src/interfaces/RestroomAvailability.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+    public abstract void readyToUse();
```
Comment (ankochergin) [inline src/interfaces/RestroomAvailability.java]: I think method name is not clear

File: src/interfaces/IEat.java [hunk 3]
```diff
@@ -1,5 +1,8 @@
 package interfaces;
 
-public interface IEat {
-    void eat();
+import airport.FoodCourt;
+
+public abstract class IEat {
```
Comment (ankochergin) [inline src/interfaces/IEat.java]: this should be an intercase too

File: src/interfaces/IAnnouncement.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package interfaces;
+
+public interface IAnnouncement {
```
Comment (ankochergin) [inline src/interfaces/IAnnouncement.java:3]: is this used anywhere?

File: src/people/Passenger.java [hunk 5]
```diff
@@ -44,10 +44,8 @@ public void setSecurity(boolean security) {
         this.security = security;
     }
 
-    @Override
-    public void eat() {
-        System.out.println("Im hungry for" + FoodCourt.foodType + "from" + FoodCourt.restaurantName + "and the price is" + FoodCourt.price);
-    }
+    if()
+
```
Comment (ankochergin) [inline src/people/Passenger.java]: is it needed here?

File: src/interfaces/IAnnouncement.java (hunk 4 above)
Comment (csestak-solvd) [reply src/interfaces/IAnnouncement.java:3]: no this hasnt been implemented anywhere. i wanted to use it in Airport. I hadnt made it that far yet. will work on that

File: src/people/Passenger.java (hunk 5 above)
Comment (csestak-solvd) [reply src/people/Passenger.java]: This was an accident. i removed it just after pushing it. oops. sorry. .good catch

**Pull request closed without merging**